### ISSUE TITLE
Question about real-time-html and time information

### ISSUE THREAD

**Issue by Danlock** [allinurl/goaccess#1184]
My nginx format:
```
log_format extended  '$remote_addr - $remote_user $host [$time_local] "$request" '
                 '$status $body_bytes_sent "$http_referer" "$http_user_agent" '
                  '$request_time $upstream_response_time';
```
My goaccess.config:
```
time-format %T
date-format %d/%b/%Y
log-format %h %^ %^ %v %^[%d:%t %^] "%r" %s %b "%R" "%u" %T %T 
```
Sample log line:
```
217.101.108.154 - - real.address.com [27/Jul/2018:13:26:14 -0400] "GET /halp/me.json HTTP/1.1" 200 44 "-" "Go-http-client/1.1" 0.800 0.800
```
My resulting HTML report is lacking time series information. I am using goaccess 1.2, on ubuntu 18.04.

**Comment by allinurl** [comment]
That format looks right to me. Could you please post the output you are getting? Thanks.

**Comment by Danlock** [comment]
Here is the html report converted to JSON.

```
{
  "general": {
    "start_date": "27/Jul/2018",
    "end_date": "27/Jul/2018",
    "date_time": "2018-07-27 14:01:59 -0400",
    "total_requests": 511,
    "valid_requests": 511,
    "failed_requests": 0,
    "generation_time": 0,
    "unique_visitors": 4,
    "unique_files": 34,
    "excluded_hits": 0,
    "unique_referrers": 9,
    "unique_not_found": 0,
    "unique_static_files": 10,
    "log_size": 0,
    "bandwidth": 1646966,
    "log_path": [
      "STDIN"
    ]
  },
  "visitors": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1646966,
        "min": 1646966
      },
      "visitors": {
        "count": 4,
        "max": 4,
        "min": 4
      },
      "hits": {
        "count": 511,
        "max": 511,
        "min": 511
      },
      "data": {
        "unique": 1
      }
    },
    "data": [
      {
        "hits": {
          "count": 511,
          "percent": 100
        },
        "visitors": {
          "count": 4,
          "percent": 100
        },
        "bytes": {
          "count": 1646966,
          "percent": 100
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "20180727"
      }
    ]
  },
  "requests": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1642152,
        "max": 1026869,
        "min": 0
      },
      "visitors": {
        "count": 34,
        "max": 2,
        "min": 1
      },
      "hits": {
        "count": 501,
        "max": 73,
        "min": 1
      },
      "data": {
        "unique": 34
      }
    },
    "data": [
      {
        "hits": {
          "count": 73,
          "percent": 14.57
        },
        "visitors": {
          "count": 2,
          "percent": 5.88
        },
        "bytes": {
          "count": 118789,
          "percent": 7.23
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/"
      },
      {
        "hits": {
          "count": 37,
          "percent": 7.39
        },
        "visitors": {
          "count": 2,
          "percent": 5.88
        },
        "bytes": {
          "count": 67304,
          "percent": 4.1
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/users/sign_in"
      },
      {
        "hits": {
          "count": 37,
          "percent": 7.39
        },
        "visitors": {
          "count": 2,
          "percent": 5.88
        },
        "bytes": {
          "count": 3824,
          "percent": 0.23
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "POST",
        "protocol": "HTTP/1.1",
        "data": "/users/sign_in"
      },
      {
        "hits": {
          "count": 36,
          "percent": 7.19
        },
        "visitors": {
          "count": 1,
          "percent": 2.94
        },
        "bytes": {
          "count": 144807,
          "percent": 8.82
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/live/grid"
      }
    ]
  },
  "static_requests": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 4814,
        "max": 1273,
        "min": 0
      },
      "visitors": {
        "count": 10,
        "max": 1,
        "min": 1
      },
      "hits": {
        "count": 10,
        "max": 1,
        "min": 1
      },
      "data": {
        "unique": 10
      }
    },
    "data": [
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 519,
          "percent": 10.78
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/icons/white_up_arrow-2e0bb17b7283687f0b5a13aa524a6338.svg"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 490,
          "percent": 10.18
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/icons/white_down_arrow-1fd3d9559a88745d2e33be5b4ef538eb.svg"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 0,
          "percent": 0
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/application-3e3c7e8eb45e483ab36ea0dcfd76bd60.css"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 0,
          "percent": 0
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/application-e0b1743293d1394280f8136f70922629.js"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 1273,
          "percent": 26.44
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/offline-bbe040078cb0a167c3313b3fc64df9da.png"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 289,
          "percent": 6
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/product_icons/broadcast-server-5af83a13202b30fa6e25e70774be28ea.png"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 699,
          "percent": 14.52
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/icons/white_check_mark-93276f370318a22c63194d9bdec39df7.svg"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 365,
          "percent": 7.58
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/icons/transfer-ec3878e3a02f5e9ae263c383dc3c69d8.svg"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 291,
          "percent": 6.04
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/icons/record-b5370867a4f22c9b8e49a8039215b70f.svg"
      },
      {
        "hits": {
          "count": 1,
          "percent": 10
        },
        "visitors": {
          "count": 1,
          "percent": 10
        },
        "bytes": {
          "count": 888,
          "percent": 18.45
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "method": "GET",
        "protocol": "HTTP/1.1",
        "data": "/assets/icons/disabled-transferring-1b0d01a3bd0fd8717e80ab6078d11b52.svg"
      }
    ]
  },
  "not_found": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "visitors": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "hits": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "data": {
        "unique": 0
      }
    },
    "data": []
  },
  "hosts": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1646966,
        "min": 1646966
      },
      "visitors": {
        "count": 4,
        "max": 4,
        "min": 4
      },
      "hits": {
        "count": 511,
        "max": 511,
        "min": 511
      },
      "data": {
        "unique": 1
      }
    },
    "data": [
      {
        "hits": {
          "count": 511,
          "percent": 100
        },
        "visitors": {
          "count": 4,
          "percent": 100
        },
        "bytes": {
          "count": 1646966,
          "percent": 100
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "[REDACTED]",
        "country": "CA Canada"
      }
    ]
  },
  "os": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1239826,
        "min": 1009
      },
      "visitors": {
        "count": 4,
        "max": 1,
        "min": 1
      },
      "hits": {
        "count": 511,
        "max": 432,
        "min": 2
      },
      "data": {
        "unique": 4
      }
    },
    "data": [
      {
        "hits": {
          "count": 432,
          "percent": 84.54
        },
        "visitors": {
          "count": 1,
          "percent": 25
        },
        "bytes": {
          "count": 392182,
          "percent": 23.81
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "Unknown",
        "items": [
          {
            "hits": {
              "count": 432,
              "percent": 84.54
            },
            "visitors": {
              "count": 1,
              "percent": 25
            },
            "bytes": {
              "count": 392182,
              "percent": 23.81
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "Unknown"
          }
        ]
      },
      {
        "hits": {
          "count": 77,
          "percent": 15.07
        },
        "visitors": {
          "count": 2,
          "percent": 50
        },
        "bytes": {
          "count": 1253775,
          "percent": 76.13
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "Macintosh",
        "items": [
          {
            "hits": {
              "count": 54,
              "percent": 10.57
            },
            "visitors": {
              "count": 1,
              "percent": 25
            },
            "bytes": {
              "count": 13949,
              "percent": 0.85
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "OS X Puma"
          },
          {
            "hits": {
              "count": 23,
              "percent": 4.5
            },
            "visitors": {
              "count": 1,
              "percent": 25
            },
            "bytes": {
              "count": 1239826,
              "percent": 75.28
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "macOS Sierra"
          }
        ]
      },
      {
        "hits": {
          "count": 2,
          "percent": 0.39
        },
        "visitors": {
          "count": 1,
          "percent": 25
        },
        "bytes": {
          "count": 1009,
          "percent": 0.06
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "Linux",
        "items": [
          {
            "hits": {
              "count": 2,
              "percent": 0.39
            },
            "visitors": {
              "count": 1,
              "percent": 25
            },
            "bytes": {
              "count": 1009,
              "percent": 0.06
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "Ubuntu"
          }
        ]
      }
    ]
  },
  "browsers": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1239826,
        "min": 14958
      },
      "visitors": {
        "count": 4,
        "max": 2,
        "min": 1
      },
      "hits": {
        "count": 511,
        "max": 432,
        "min": 23
      },
      "data": {
        "unique": 3
      }
    },
    "data": [
      {
        "hits": {
          "count": 432,
          "percent": 84.54
        },
        "visitors": {
          "count": 1,
          "percent": 25
        },
        "bytes": {
          "count": 392182,
          "percent": 23.81
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "Others",
        "items": [
          {
            "hits": {
              "count": 432,
              "percent": 84.54
            },
            "visitors": {
              "count": 1,
              "percent": 25
            },
            "bytes": {
              "count": 392182,
              "percent": 23.81
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "Go-http-client/1.1"
          }
        ]
      },
      {
        "hits": {
          "count": 79,
          "percent": 15.46
        },
        "visitors": {
          "count": 3,
          "percent": 75
        },
        "bytes": {
          "count": 1254784,
          "percent": 76.19
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "Chrome",
        "items": [
          {
            "hits": {
              "count": 56,
              "percent": 10.96
            },
            "visitors": {
              "count": 2,
              "percent": 50
            },
            "bytes": {
              "count": 14958,
              "percent": 0.91
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "Chrome/67.0.3396.99"
          },
          {
            "hits": {
              "count": 23,
              "percent": 4.5
            },
            "visitors": {
              "count": 1,
              "percent": 25
            },
            "bytes": {
              "count": 1239826,
              "percent": 75.28
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "Chrome/63.0.3239.84"
          }
        ]
      }
    ]
  },
  "visit_time": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1614122,
        "min": 32844
      },
      "visitors": {
        "count": 6,
        "max": 4,
        "min": 2
      },
      "hits": {
        "count": 511,
        "max": 464,
        "min": 47
      },
      "data": {
        "unique": 2
      }
    },
    "data": [
      {
        "hits": {
          "count": 464,
          "percent": 90.8
        },
        "visitors": {
          "count": 4,
          "percent": 66.67
        },
        "bytes": {
          "count": 1614122,
          "percent": 98.01
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "13"
      },
      {
        "hits": {
          "count": 47,
          "percent": 9.2
        },
        "visitors": {
          "count": 2,
          "percent": 33.33
        },
        "bytes": {
          "count": 32844,
          "percent": 1.99
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "14"
      }
    ]
  },
  "vhosts": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1646966,
        "min": 1646966
      },
      "visitors": {
        "count": 4,
        "max": 4,
        "min": 4
      },
      "hits": {
        "count": 511,
        "max": 511,
        "min": 511
      },
      "data": {
        "unique": 1
      }
    },
    "data": [
      {
        "hits": {
          "count": 511,
          "percent": 100
        },
        "visitors": {
          "count": 4,
          "percent": 100
        },
        "bytes": {
          "count": 1646966,
          "percent": 100
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "real.address.com"
      }
    ]
  },
  "referrers": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1141282,
        "min": 2823
      },
      "visitors": {
        "count": 13,
        "max": 3,
        "min": 1
      },
      "hits": {
        "count": 511,
        "max": 363,
        "min": 3
      },
      "data": {
        "unique": 9
      }
    },
    "data": [
      {
        "hits": {
          "count": 363,
          "percent": 71.04
        },
        "visitors": {
          "count": 3,
          "percent": 23.08
        },
        "bytes": {
          "count": 222456,
          "percent": 13.51
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "-"
      },
      {
        "hits": {
          "count": 40,
          "percent": 7.83
        },
        "visitors": {
          "count": 2,
          "percent": 15.38
        },
        "bytes": {
          "count": 154437,
          "percent": 9.38
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "https://real.address.com/users/sign_in"
      },
      {
        "hits": {
          "count": 36,
          "percent": 7.05
        },
        "visitors": {
          "count": 1,
          "percent": 7.69
        },
        "bytes": {
          "count": 65496,
          "percent": 3.98
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "https://real.address.com"
      },
      {
        "hits": {
          "count": 33,
          "percent": 6.46
        },
        "visitors": {
          "count": 1,
          "percent": 7.69
        },
        "bytes": {
          "count": 3069,
          "percent": 0.19
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "https://real.address.com/gateways/749266"
      },
      {
        "hits": {
          "count": 20,
          "percent": 3.91
        },
        "visitors": {
          "count": 1,
          "percent": 7.69
        },
        "bytes": {
          "count": 6854,
          "percent": 0.42
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "https://real.address.com/"
      },
      {
        "hits": {
          "count": 6,
          "percent": 1.17
        },
        "visitors": {
          "count": 2,
          "percent": 15.38
        },
        "bytes": {
          "count": 3252,
          "percent": 0.2
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "https://real.address.com/assets/application-3e3c7e8eb45e483ab36ea0dcfd76bd60.css"
      },
      {
        "hits": {
          "count": 6,
          "percent": 1.17
        },
        "visitors": {
          "count": 1,
          "percent": 7.69
        },
        "bytes": {
          "count": 1141282,
          "percent": 69.3
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "https://real.address.com/clips"
      },
      {
        "hits": {
          "count": 4,
          "percent": 0.78
        },
        "visitors": {
          "count": 1,
          "percent": 7.69
        },
        "bytes": {
          "count": 47297,
          "percent": 2.87
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "https://real.address.com/notifications"
      },
      {
        "hits": {
          "count": 3,
          "percent": 0.59
        },
        "visitors": {
          "count": 1,
          "percent": 7.69
        },
        "bytes": {
          "count": 2823,
          "percent": 0.17
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "https://real.address.com/emergency-contacts"
      }
    ]
  },
  "referring_sites": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1424510,
        "max": 1424510,
        "min": 1424510
      },
      "visitors": {
        "count": 4,
        "max": 4,
        "min": 4
      },
      "hits": {
        "count": 148,
        "max": 148,
        "min": 148
      },
      "data": {
        "unique": 1
      }
    },
    "data": [
      {
        "hits": {
          "count": 148,
          "percent": 100
        },
        "visitors": {
          "count": 4,
          "percent": 100
        },
        "bytes": {
          "count": 1424510,
          "percent": 100
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "real.address.com"
      }
    ]
  },
  "keyphrases": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "visitors": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "hits": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "data": {
        "unique": 0
      }
    },
    "data": []
  },
  "status_codes": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1638443,
        "min": 0
      },
      "visitors": {
        "count": 7,
        "max": 4,
        "min": 1
      },
      "hits": {
        "count": 511,
        "max": 429,
        "min": 2
      },
      "data": {
        "unique": 4
      }
    },
    "data": [
      {
        "hits": {
          "count": 429,
          "percent": 83.95
        },
        "visitors": {
          "count": 4,
          "percent": 57.14
        },
        "bytes": {
          "count": 1638443,
          "percent": 99.48
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "2xx Success",
        "items": [
          {
            "hits": {
              "count": 429,
              "percent": 83.95
            },
            "visitors": {
              "count": 4,
              "percent": 57.14
            },
            "bytes": {
              "count": 1638443,
              "percent": 99.48
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "200 - OK: The request sent by the client was successful"
          }
        ]
      },
      {
        "hits": {
          "count": 76,
          "percent": 14.87
        },
        "visitors": {
          "count": 3,
          "percent": 42.86
        },
        "bytes": {
          "count": 8523,
          "percent": 0.52
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "3xx Redirection",
        "items": [
          {
            "hits": {
              "count": 74,
              "percent": 14.48
            },
            "visitors": {
              "count": 2,
              "percent": 28.57
            },
            "bytes": {
              "count": 8523,
              "percent": 0.52
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "302 - Moved Temporarily (redirect)"
          },
          {
            "hits": {
              "count": 2,
              "percent": 0.39
            },
            "visitors": {
              "count": 1,
              "percent": 14.29
            },
            "bytes": {
              "count": 0,
              "percent": 0
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "304 - Not Modified: Resource has not been modified"
          }
        ]
      },
      {
        "hits": {
          "count": 6,
          "percent": 1.17
        },
        "visitors": {
          "count": 0,
          "percent": 0
        },
        "bytes": {
          "count": 0,
          "percent": 0
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "4xx Client Error",
        "items": [
          {
            "hits": {
              "count": 6,
              "percent": 1.17
            },
            "visitors": {
              "count": 0,
              "percent": 0
            },
            "bytes": {
              "count": 0,
              "percent": 0
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "499 - (Nginx) Connection closed by client while processing request"
          }
        ]
      }
    ]
  },
  "geolocation": {
    "metadata": {
      "avgts": {
        "avg": 0
      },
      "cumts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "maxts": {
        "count": 0,
        "max": 0,
        "min": 0
      },
      "bytes": {
        "count": 1646966,
        "max": 1646966,
        "min": 1646966
      },
      "visitors": {
        "count": 4,
        "max": 4,
        "min": 4
      },
      "hits": {
        "count": 511,
        "max": 511,
        "min": 511
      },
      "data": {
        "unique": 1
      }
    },
    "data": [
      {
        "hits": {
          "count": 511,
          "percent": 100
        },
        "visitors": {
          "count": 4,
          "percent": 100
        },
        "bytes": {
          "count": 1646966,
          "percent": 100
        },
        "avgts": 0,
        "cumts": 0,
        "maxts": 0,
        "data": "NA North America",
        "items": [
          {
            "hits": {
              "count": 511,
              "percent": 100
            },
            "visitors": {
              "count": 4,
              "percent": 100
            },
            "bytes": {
              "count": 1646966,
              "percent": 100
            },
            "avgts": 0,
            "cumts": 0,
            "maxts": 0,
            "data": "CA Canada"
          }
        ]
      }
    ]
  }
}
```

**Comment by Danlock** [comment]
I now realize that the real-time-html option is whats stripping out the time statistics.
Is there no way to get at least Max T.S. with that option on?

**Comment by allinurl** [comment]
Is it only when you export the JSON output from the HTML output or is empty in the HTML report? Are you in v1.2? For me this looks more like a format issue, especially since https://rt.goaccess.io is just working fine.

**Comment by Danlock** [comment]
It is empty in the HTML report and in the JSON when I set --real-time-html true
with --real-time-html false the time data appears .

**Comment by Danlock** [comment]
To help you track down the issue this is how i am using goaccess.

With real-time-html:

`ssh root@super.real.address 'tail -f -n +0 /var/log/nginx/extended.log' |  goaccess --time-format %T --date-format %d/%b/%Y --log-format "%h %^ %^ %v %^[%d:%t %^] \"%r\" %s %b \"%R\" \"%u\" %^ %T" -o ./report.html --real-time-html -
`

Without:

`ssh root@super.real.address 'tail -n +0 /var/log/nginx/extended.log' |  goaccess --time-format %T --date-format %d/%b/%Y --log-format "%h %^ %^ %v %^[%d:%t %^] \"%r\" %s %b \"%R\" \"%u\" %^ %T" -o ./report.html  -
`

**Comment by Danlock** [comment]
And in regards to the format, the sample log line I initially posted is pretty representative. It's not like most of them are missing time information or anything. If that was the case, that wouldn't explain why it works with real-time-html turned off.

**Comment by allinurl** [comment]
This has been fixed as of v1.4.3. Thanks again for reporting it.

Closing this. Feel free to reopen it if needed.
